charts/metering-ansible-operator/upstream-values.yaml

```diff
@@ -0,0 +1,515 @@
+            },
+            "spec": {
+              "catalog": "hive",
+              "columns": [
+                {
+                  "name": "cost_per_gigabyte_hour",
+                  "type": "double"
+                },
+                {
+                  "name": "cost_per_cpu_hour",
+                  "type": "double"
+                },
+                {
+                  "name": "currency",
+                  "type": "varchar"
+                }
+              ],
+              "createTableAs": true,
+              "query": "SELECT * FROM (\n  VALUES (10.00, 50.00, 'USD')\n) AS t (cost_per_gigabyte_hour, cost_per_cpu_hour, currency)\n",
+              "schema": "default",
+              "tableName": "example_baremetal_cost"
+            }
+          },
+          {
+            "apiVersion": "metering.openshift.io/v1",
+            "kind": "HiveTable",
+            "metadata": {
+              "name": "apache-log",
+              "annotations": {
+                "reference": "based on the RegEx example from https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-RowFormats&SerDe"
+              }
+            },
+            "spec": {
+              "columns": [
+                {
+                  "name": "host",
+                  "type": "string"
+                },
+                {
+                  "name": "identity",
+                  "type": "string"
+                },
+                {
+                  "name": "user",
+                  "type": "string"
+                },
+                {
+                  "name": "time",
+                  "type": "string"
+                },
+                {
+                  "name": "request",
+                  "type": "string"
+                },
+                {
+                  "name": "status",
+                  "type": "string"
+                },
+                {
+                  "name": "size",
+                  "type": "string"
+                },
+                {
+                  "name": "referer",
+                  "type": "string"
+                },
+                {
+                  "name": "agent",
+                  "type": "string"
+                }
+              ],
+              "databaseName": "default",
+              "external": true,
+              "fileFormat": "TEXTFILE",
+              "location": "s3a://my-bucket/apache_logs",
+              "rowFormat": "SERDE 'org.apache.hadoop.hive.serde2.RegexSerDe'\nWITH SERDEPROPERTIES (\n  \"input.regex\" = \"([^ ]*) ([^ ]*) ([^ ]*) (-|\\\\[[^\\\\]]*\\\\]) ([^ \\\"]*|\\\"[^\\\"]*\\\") (-|[0-9]*) (-|[0-9]*)(?: ([^ \\\"]*|\\\"[^\\\"]*\\\") ([^ \\\"]*|\\\"[^\\\"]*\\\"))?\"\n)\n",
+              "tableName": "apache_log"
+            }
+          }
+        ]
+
+  subscriptionName: metering
+  subscriptionChannel: stable
+  subscriptionCatalogSource: redhat-operators
+  subscriptionCatalogSourceNamespace: openshift-marketplace
```

Comment:
Needs to be modified, and figure out what the correct value is. Not sure if/how this is installed when using OLM on upstream.